README.md

```diff
@@ -45,17 +45,20 @@ The document number will be read from the files by default.
 # - output (echo) the commands that would be applied without noop
 # Good for testing
 docsplit.sh --noop "$DIR/data/Simple.pdf" "$TEST_DIR/file"
```

Comment:
It could be helpful to provide the test data `/data/Simple.pdf` together with this project, so that people can directly try it out?